pkg/kubelet/server/portforward/httpstream.go

```diff
@@ -0,0 +1,309 @@
+/*
```

Comment:
Not really, it originally started as portforward.go but portforward.go was reimplemented so you won't see it as a move. I could rename it in the original commit so it doesn't change as much but it does get affected because of ServePortForward.